src/sagemaker/session.py

```diff
@@ -3402,13 +3417,6 @@ def create_model(
             default_value=False,
             sagemaker_session=self,
         )
```

Comment:
Consider adding a comment in create_model that Env is injected from the config in the helper method? (just so someone doesnt think it was a miss)

Reply:
Now there is no need of this since we are injecting Env config in the create_model function itself.

Reply:
done